config/site/src/getting-started.md

```diff
@@ -181,10 +181,10 @@ can be used in your publishing system to validate the indexing payloads or for c
 
 ## Resources
 
-- **[ElasticGraph Query API Documentation]({% link query-api/overview.md %})**
-- **[ElasticGraph Ruby API Documentation]({{ '/docs/' | append: site.data.doc_versions.latest_version | relative_url }})**
+- **[ElasticGraph Query API Documentation]({% link query-api.md %})**
+- **[ElasticGraph Ruby API Documentation]({% link api-docs.md %})**
+- **[ElasticGraph Guides]({% link guides.md %})**
 - **[GraphQL Introduction](https://graphql.org/learn/)**
```

Comment:
Let's re order these based on the order we think the average new engineer should visit these:

* Maybe GraphQL intro first? That's the most fundamental here...
* Then follow the order of our nav? (Query API, Guides, API Docs)?